src/citrine/resources/predictor_evaluation_workflow.py

```diff
@@ -53,3 +54,36 @@ def delete(self, uid: Union[UUID, str]) -> Response:
         """Predictor Evaluation Workflows cannot be deleted; they can be archived instead."""
         raise NotImplementedError(
             "Predictor Evaluation Workflows cannot be deleted; they can be archived instead.")
+
+    def create_default(self, predictor_id: UUID) -> PredictorEvaluationWorkflow:
```

Comment:
@maxhutch What about something like what I've written below. I'm worried these docs might get out of date, so I'm open to suggestions on what you think could go here instead.

Reply:
Looks good!